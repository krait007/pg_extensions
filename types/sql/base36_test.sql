CREATE EXTENSION base36;
CREATE TABLE base36_test(val base36);
INSERT INTO base36_test VALUES ('123'), ('3c'), ('5A'), ('zZz');
SELECT * FROM base36_test;
SELECT * FROM base36_test ORDER BY val;
