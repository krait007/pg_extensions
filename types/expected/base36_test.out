CREATE EXTENSION base36;
CREATE TABLE base36_test(val base36);
INSERT INTO base36_test VALUES ('123'), ('3c'), ('5A'), ('zZz');
SELECT * FROM base36_test;
 val 
-----
 123
 3c
 5a
 zzz
(4 rows)

SELECT * FROM base36_test ORDER BY val;
 val 
-----
 3c
 5a
 123
 zzz
(4 rows)

